packages/experiments/src/components/Pagination/PageNumber.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { IPageNumberProps } from './PageNumber.types';
+
+export class PageNumber extends React.Component<IPageNumberProps, {}> {
+  constructor(props: IPageNumberProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const { className, ariaLabel, page, selected } = this.props;
+
+    return (
+      <button
+        key={page}
+        className={className}
+        onClick={this.onClick}
+        aria-selected={selected}
+        aria-label={ariaLabel}
+      >
+        {page}
+      </button>
+    );
+  }
+
+  private onClick = () => {
+    this.props.applyPage(this.props.page - 1);
```

Comment:
I don't think we should do the page number to index conversion for consumers of this control. If the `props.page` expected a 1-based numbering system, then I think the operations should expect the same 1-based number